File: Backend/CaffShop.Shared/DTOs/CommentDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CaffShop.Shared.DTOs
{
  public class CommentDto
  {
    public int Id { get; set; }
    public DateTime CreatedDate { get; set; }
    public string Content { get; set; }
    public string UserName { get; set; }
    public bool IsOwner { get; set; }
  }
}

File: Frontend/lib/models/comment_dto.dart
class CommentDto {
  final int id;
  final String createdDate;
  final String content;
  final String userName;
  final bool isOwner;

  CommentDto(
      {required this.id,
      required this.createdDate,
      required this.content,
      required this.userName,
      required this.isOwner});

  factory CommentDto.fromJson(Map<String, dynamic> json) => CommentDto(
      id: json['id'],
      createdDate: json['createdDate'],
      content: json['content'],
      userName: json['userName'],
      isOwner: json['isOwner']);
}
